modules/custom/d_content/d_content.module

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * @file
+ * D_content module.
+ */
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function d_content_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  if ($form_id == 'node_content_page_edit_form' || $form_id == 'node_content_page_form') {
+    if (isset($form['field_header_cta'])) {
+      $form['field_header_cta']['#states'] = [
+        'visible' => [
+          ':input[name="field_header_layout"]' => ['value' => 'header_with_cta'],
+        ],
+      ];
+    }
+  }
```

Comment:
zobacz PR od @jakub-wozniak-droptica on tam uprościł trochę ten alter, mógłbyś proszę zrobić analogicznie ?
https://github.com/droptica/droopler/pull/573/files

Reply:
@rafal-sereda-droptica a jak chciałbyś to uprościć?

Reply:
aaaa źle, pośpieszyłem się i nie popatrzyłem dokładnie... chodziło mi o dodanie `btn btn-primary` tak jak to Kuba robił u siebie w PR

Reply:
Dodałem nowego hooka tak żeby dodawał btn btn-primary domyślnie dla pola CTA